apps/mobile/src/app/components/Auctions/AuctionsPage.tsx

```diff
@@ -37,16 +41,26 @@ export function AuctionsPage() {
   }, [sortBy, sortDir, selectedCategoryID]);
 
   return (
-    <AuctionsTemplate
-      categories={categories}
-      selectedCategoryID={selectedCategoryID}
-      onCategoryChange={(id) => setSelectedCategoryID(id)}
-      sortBy={sortBy}
-      onSortByChange={(sortBy) => setSortBy(sortBy)}
-      sortDir={sortDir}
-      onSortDirChange={(sortDir) => setSortDir(sortDir)}
-      auctions={auctions}
-    />
+    <View style={{ flex: 1 }}>
+      <ScrollView>
+        <AuctionsTemplate
+          categories={categories}
+          selectedCategoryID={selectedCategoryID}
+          onCategoryChange={(id) => setSelectedCategoryID(id)}
+          sortBy={sortBy}
+          onSortByChange={(sortBy) => setSortBy(sortBy)}
+          sortDir={sortDir}
+          onSortDirChange={(sortDir) => setSortDir(sortDir)}
+          auctions={auctions}
+        />
+      </ScrollView>
+      <FooterMenu
```

Comment:
We should use **page -> template** approach, but in this case i couldn't find any template for FooterMenu (or something similar)

Reply:
FooterMenu can be only as organism. After that you can add this to auctionPageTemplate.